chapters/interface.tex

```diff
@@ -308,67 +307,67 @@ \subsection{Inheritance Interface or Class Type}\doublelabel{inheritance-interfa
     Whether it is a built-in type and the built-in type (\lstinline!RealType!,
     \lstinline!IntegerType!, \lstinline!StringType! or \lstinline!BooleanType!).
   \item
-    Visibility (public or protected).
+    Visibility (\lstinline!public! or \lstinline!protected!).
   \end{itemize}
 \end{itemize}
 
 \section{Interface Compatibility or Subtyping}\doublelabel{interface-compatibility-or-subtyping}
 
-An interface of a class or component A is compatible with an interface
-of a class or component B (or the constraining interface of B), or
-equivalently that the type of A is a subtype of the type of B, iff
-{[}\emph{intuitively all important elements of B must be present in A}
+An interface of a class or component \lstinline!A! is compatible with an interface
+of a class or component \lstinline!B! (or the constraining interface of \lstinline!B!), or
+equivalently that the type of \lstinline!A! is a subtype of the type of \lstinline!B!, iff
+{[}\emph{intuitively all important elements of \lstinline!B! must be present in \lstinline!A!}
 {]}\emph{:}
 
 \begin{itemize}
 \item
-  A is a class if and only if B is a class (and thus: A is a component
-  if and only if B is a component).
+  \lstinline!A! is a class if and only if \lstinline!B! is a class (and thus: \lstinline!A! is a component
+  if and only if \lstinline!B! is a component).
 \item
-  If A has an \lstinline!operator record! base-class then B must also have one and
-  it must be the same. If A does not have an operator record base-class
-  then B may not have one. See \autoref{overloaded-operators}.
+  If A has an \lstinline!operator record! base-class then \lstinline!B! must also have one and
```

Comment:
The first A should also be in lstinline, right?

Reply:
Yes.  Fixed.